deleteDiscordMessages.js

```diff
@@ -211,6 +213,10 @@
             const myMessages = data.messages.map(convo => convo.find(message => message.hit===true));
             const systemMessages = myMessages.filter(msg => msg.type !== 0); // https://discord.com/developers/docs/resources/channel#message-object-message-types
             const deletableMessages = myMessages.filter(msg => msg.type === 0);
+            const messagesToDelete = deletableMessages.filter(msg => {
```

Comment:
This could be `const messagesToDelete = deletableMessages.filter(msg => (includePinned || !msg.pinned));` but I decided to implement it in a more extendable form ¯\_(ツ)_/¯